wasm/validator/derive/src/entrypoint.rs

```diff
@@ -66,12 +66,8 @@ pub fn impl_entrypoint(attr: TokenStream, item: TokenStream) -> TokenStream {
 
     match &fn_item.sig.ident {
         fn_name if fn_name == "validate" => impl_validate_entrypoint(attr, fn_item),
-        fn_name if fn_name == "permission_token_schema" => {
-            impl_permission_token_schema_entrypoint(&attr, fn_item)
-        }
-        _ => panic!(
-            "Validator entrypoint name should be either `validate` or `permission_token_schema`"
-        ),
+        fn_name if fn_name == "migrate" => impl_migrate_entrypoint(&attr, fn_item),
+        _ => panic!("Validator entrypoint name should be either `validate` or `migrate`"),
```

Comment:
Actually, it is possible to define entrypoints names (`"validate"`, `"migrate"`, `"permission_tokens"`) as consts.

Thus:

- It is possible to re-use them in the panic message here;
- If moved to a separate crate, it is possible to mention then in doc comments and avoid pasting everywhere `migrate()` and `validate()`.

The outcome is a more coherent and refactoring-prone codebase and documentation.

Reply:
refactoring-prone usually means that you want to refactor it often. You probably mean easy-to-refactor.

Reply:
Created #3757